### PR TITLE
Include a message with the UI link when commenting back to the PR

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: cmd/payload-testing-prow-plugin/server.go [hunk 1]
```diff
@@ -229,6 +236,9 @@ func (s *server) handle(l *logrus.Entry, ic github.IssueCommentEvent) string {
 		}
 		messages = append(messages, message(spec, jobNames))
 	}
+
+	messages = append(messages, fmt.Sprintf("See details on %s/%s/%s\n", prPayloadTestsUIURL, builder.namespace, prpqrName))
```
Comment (bbguimaraes) [inline cmd/payload-testing-prow-plugin/server.go:240]: This has to be done for each `spec`.

**Review by bbguimaraes — CHANGES_REQUESTED** [review]

File: cmd/payload-testing-prow-plugin/server.go (hunk 1 above)
Comment (droslean) [reply cmd/payload-testing-prow-plugin/server.go:240]: Why? The prpqr name will be the same. In the link the user can see all the jobs then.

Comment (bbguimaraes) [reply cmd/payload-testing-prow-plugin/server.go:240]: `specs` can have multiple items if the comment being processed has more than one `/payload` command.

Comment (petr-muller) [reply cmd/payload-testing-prow-plugin/server.go:240]: yes, and `prpqr` names will be different (they must be, they are kube resources), the builder is adding counter-based suffixes `-0` to the names

Comment (droslean) [reply cmd/payload-testing-prow-plugin/server.go]: Yea got it! Updated.

I left the `trigger 0 jobs ` message untouched for now

**Pull request merged**